recipes/micro-step-helpers/split-string/cpp.md

```diff
@@ -0,0 +1,48 @@
+{{< Draft >}}
+
+# Split String
+
+
+You have a function you would like to add a return at the end of:
```

Comment:
Looks like copy-and-paste error from your base recipe. Please update for this recipe.